openapi.yaml

```diff
@@ -794,8 +843,8 @@ components:
     SuccessResponse:
       type: object
       properties:
-        status:
-          type: string
+        tweet_id:
```

Comment:
this is still valid https://github.com/mo-pm888/twitter/pull/43#discussion_r1307658185 (not fixed)

Reply:
better, but the names are terrible